app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception, unless: -> { request.format.json? }
+
+  def hogehoge
+    aa=111  
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/UselessAssignment: Useless assignment to variable - aa.